app/bundles/LeadBundle/Command/SegmentStatCommand.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Mautic\LeadBundle\Command;
+
+use Mautic\CoreBundle\Command\ModeratedCommand;
+use Mautic\CoreBundle\Helper\CoreParametersHelper;
+use Mautic\CoreBundle\Helper\PathsHelper;
+use Mautic\LeadBundle\Event\GetStatDataEvent;
+use Mautic\LeadBundle\LeadEvents;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\EventDispatcher\EventDispatcherInterface;
+
+class SegmentStatCommand extends ModeratedCommand
+{
+    public function __construct(
+        private EventDispatcherInterface $dispatcher,
+        PathsHelper $pathsHelper,
+        CoreParametersHelper $coreParametersHelper,
+    ) {
+        parent::__construct($pathsHelper, $coreParametersHelper);
+    }
+
+    protected function configure(): void
+    {
+        $this
+            ->setName('mautic:segments:stat')
+            ->setDescription('Gather Segment Statistics');
+
+        parent::configure();
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $io         = new SymfonyStyle($input, $output);
+        $event      = new GetStatDataEvent();
+        $this->dispatcher->dispatch($event, LeadEvents::LEAD_LIST_STAT);
```

Comment:
The event name is optional in newer Symfony versions. Let's go without it and use the event class in the subscribers instead.
```suggestion
        $this->dispatcher->dispatch($event);
```